### ISSUE TITLE
G6 - Global Navbar dropdown menu dividers do not convey meaning to webreaders

### ISSUE THREAD

**Issue by enichol** [brharp/hjckrrh#220]
![image](https://cloud.githubusercontent.com/assets/14219820/9936293/3a4a0310-5d29-11e5-8ee5-5a34007ca2ab.png)

Currently the dividers between subgroups on Global Navbar dropdowns do not convey any meaning through a webreader. To visual users they indicate that the links in the list are divided into like-subgroups, but this meaning would be lost to a non-visual user. 
Currently, browsers are treating the divider differently:
-Chrome: Ignored dividers completely
-Firefox: Announces the divider as 'blank'. This may be confusing to anybody using a webreader, as it might be confused as a broken/missing link, or as a jump to the next navbar item
-IE: Ignores dividers
